### DISCUSSION TITLE
可能是因为某些盘新网络环境打不开了，进度网页的时候提示“获取设置失败： 请稍候，正在加载储存”，想进入后台关闭相关的盘也不行。

### DISCUSSION THREAD

**Discussion by yangzhenwei777** [AlistGo/alist#7206] [Q&A] [answered]
网页地址永远只显示这几个字，这个要怎么解决呢？除了等之外没别的办法了吗？

**Comment by anwen-anyi** [comment, marked as answer]
通过命令行将驱动停止，要么等待几分钟让他重试失败就好了
- https://alist.nn.ci/zh/guide/install/manual.html#%E4%B8%80%E7%9B%B4%E5%9C%A8%E5%8A%A0%E8%BD%BD%E6%80%8E%E4%B9%88%E5%8A%9E
